coq-bignums.dev fails with coq.dev since 2019-11-02

**Kind:** bug

Last known successful build:

```
BUILD_DATE=2019-11-01T03:56:55Z COQ_COMMIT=6694a1811dc4e961a81fb4464cf5aaf05f1b5752
```

Currently failing build:

`coq-bignums.dev` with https://github.com/coq/coq/tree/b0fe318ea3b54f6f5f4c911bfb0e8523405e8c5c

<details>
<summary>Docker Hub build log</summary>

```
[…]
-> compiled coq.dev
-> installed coq.dev
Processing 11/12: [coq-bignums: dune build]
+ /home/coq/.opam/4.07.1+flambda/bin/dune "build" "-p" "coq-bignums" "-j" "2" (CWD=/home/coq/.opam/4.07.1+flambda/.opam-switch/build/coq-bignums.dev)
- ocamlc plugin/.bignums_syntax_plugin.objs/byte/bignums_syntax_plugin__Bignums_syntax.{cmi,cmo,cmt}
- File "plugin/bignums_syntax.ml", line 47, characters 2-47:
- Warning 3: deprecated: ConstructRef
- Use Names.GlobRef.ConstructRef
- File "plugin/bignums_syntax.ml", line 55, characters 15-42:
- Warning 3: deprecated: ConstructRef
- Use Names.GlobRef.ConstructRef
- File "plugin/bignums_syntax.ml", line 56, characters 15-42:
- Warning 3: deprecated: ConstructRef
- Use Names.GlobRef.ConstructRef
- File "plugin/bignums_syntax.ml", line 65, characters 14-41:
- Warning 3: deprecated: ConstructRef
- Use Names.GlobRef.ConstructRef
- ocamlopt plugin/.bignums_syntax_plugin.objs/native/bignums_syntax_plugin__Bignums_syntax.{cmx,o}
- File "plugin/bignums_syntax.ml", line 47, characters 2-47:
- Warning 3: deprecated: ConstructRef
- Use Names.GlobRef.ConstructRef
- File "plugin/bignums_syntax.ml", line 55, characters 15-42:
- Warning 3: deprecated: ConstructRef
- Use Names.GlobRef.ConstructRef
- File "plugin/bignums_syntax.ml", line 56, characters 15-42:
- Warning 3: deprecated: ConstructRef
- Use Names.GlobRef.ConstructRef
- File "plugin/bignums_syntax.ml", line 65, characters 14-41:
- Warning 3: deprecated: ConstructRef
- Use Names.GlobRef.ConstructRef
- Error: coqdep returned invalid output for default/BigN/NMake.v / [phase:
- line]
- Error: coqdep returned invalid output for default/BigN/NMake_gen.v / [phase:
- line]
- Error: coqdep returned invalid output for default/BigN/Nbasic.v / [phase:
- line]
- Error: coqdep returned invalid output for default/BigNumPrelude.v / [phase:
- line]
- Error: coqdep returned invalid output for default/BigQ/BigQ.v / [phase: line]
- Error: coqdep returned invalid output for default/BigZ/BigZ.v / [phase: line]
- Error: coqdep returned invalid output for default/BigQ/QMake.v / [phase:
- line]
- Error: coqdep returned invalid output for default/BigZ/ZMake.v / [phase:
- line]
- Error: coqdep returned invalid output for default/CyclicDouble/DoubleAdd.v /
- [phase: line]
- Error: coqdep returned invalid output for default/CyclicDouble/DoubleBase.v /
- [phase: line]
- Error: coqdep returned invalid output for default/CyclicDouble/DoubleCyclic.v
- / [phase: line]
- Error: coqdep returned invalid output for default/CyclicDouble/DoubleDivn1.v
- / [phase: line]
- Error: coqdep returned invalid output for default/CyclicDouble/DoubleDiv.v /
- [phase: line]
- Error: coqdep returned invalid output for default/CyclicDouble/DoubleLift.v /
- [phase: line]
- Error: coqdep returned invalid output for default/CyclicDouble/DoubleMul.v /
- [phase: line]
- Error: coqdep returned invalid output for default/CyclicDouble/DoubleSub.v /
- [phase: line]
- Error: coqdep returned invalid output for default/CyclicDouble/DoubleSqrt.v /
- [phase: line]
- Error: coqdep returned invalid output for default/SpecViaQ/QSig.v / [phase:
- line]
- Error: coqdep returned invalid output for default/SpecViaZ/NSig.v / [phase:
- line]
- Error: coqdep returned invalid output for default/SpecViaZ/NSigNAxioms.v /
- [phase: line]
- Error: coqdep returned invalid output for default/SpecViaZ/ZSig.v / [phase:
- line]
- Error: coqdep returned invalid output for default/BigN/BigN.v / [phase: line]
- Error: coqdep returned invalid output for default/SpecViaZ/ZSigZAxioms.v /
- [phase: line]
[ERROR] The compilation of coq-bignums failed at "/home/coq/.opam/4.07.1+flambda/bin/dune build -p coq-bignums -j 2".
```

</details>

Maybe this comes from a breaking change from [these commits](https://github.com/coq/coq/compare/6694a1811dc4e961a81fb4464cf5aaf05f1b5752...b0fe318ea3b54f6f5f4c911bfb0e8523405e8c5c), but I didn't figure out yet which one…

Ideas on how to fix this? Cc @ppedrot @maximedenes

[Note that this issue is currently *blocking* for the `coqorg/coq:dev` Docker Hub nightly build]

--- a/coq-bignums.opam
+++ b/coq-bignums.opam
@@ -13,12 +13,12 @@ Provides BigN, BigZ, BigQ that used to be part of Coq standard library
 
 version: "dev"
 
-build: [make "-j%{jobs}%"]
-install: [make "install"]
+build: ["dune" "build" "-p" name "-j" jobs]
 
 depends: [
   "ocaml"
   "coq" {= "dev"}
+  "dune" {>= "1.9.0"}
 ]
 
 tags: [

